maps/randomvaults/objects.dm

```diff
@@ -76,3 +78,12 @@
 
 /obj/machinery/atmospherics/binary/msgs/rust_vault
 	name = "\improper Magnitno Priostanovleno Blok Khraneniya Gaza"
+
+/obj/item/weapon/paper/iou
+	name = "paper- 'IOU'"
+	info = "I owe you a rod of destruction. Redeemable at Milliway's at the end of time."
+
+/obj/item/weapon/gun/projectile/russian/empty/New()
+	..()
+	loaded.Cut()
```

Comment:
move this to code/modules/projectiles/guns/projectile/revolver.dm (because it doesn't need to be vault specific, and makes sense to have it with the revolver code) and change it to
```
/obj/item/weapon/gun/projectile/russian/empty/New()
..()
 for(var/obj/item/ammo_casing/AC in loaded)
  loaded -= AC
  loaded += null
  qdel(AC)